### ISSUE TITLE
Feature Request: switch for number.luxtronik2_dhw_thermal_desinfection_target

### ISSUE THREAD

**Issue by apf-doit** [BenPru/luxtronik#263]
hi benpru, 

I would like to request to get a sensor where I can turn on/off thermal desinfection, if possible. The integration gives me the target temperature but no switch since on the device it comes with another timebased menu to choose which day of the week it should run.
With an older AI LWD 70 there is no fancy new webclient for me showing the physical menu since AI obvisiouly has stopped newer firmware for my heat pump. I am on V2.88.3. Having such a menu it would be easy to use from remote, but i have to stay in front of it to turn it on/off. The reason for this feature request is simply to turn thermal desinfection on when having a lot of solar power left which I don't want to waste buy selling it to the grid.

cheers
alex

**Comment by BenPru** [comment]
Thermal desinfection is just a normal heating mode.
You can create a ha automation, script or action which activate the dhw mode and set a high number.luxtronik2_dhw_target_temperature. That's it.

**Comment by apf-doit** [comment]
yeah, that seems to be an easy way to accomplish that while thinking about it. i just thought i miss the correct sensor cuz i stucked in thinking in menu style and not heating_mode style ;)
thank you for the input.
